packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionInvoke.ts

```diff
@@ -6,6 +6,38 @@
  */
 
 /**
- * Executes sfdx evergreen:function:invoke http://localhost:8080 --payload='@functions/MyFunction/payload.json'
+ * Executes sfdx evergreen:function:invoke http://localhost:8080 --payload=@functions/MyFunction/payload.json
  */
-export async function forceFunctionInvoke() {}
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { Uri } from 'vscode';
+import { nls } from '../../messages';
+import {
+  FilePathGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from '../util';
+
+export class ForceFunctionInvoke extends SfdxCommandletExecutor<string> {
+  public build(payloadUri: string): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(nls.localize('force_function_invoke_text'))
+      .withArg('evergreen:function:invoke')
+      .withArg('http://localhost:8080')
```

Comment:
Ideally we want this port from start. We have a dependency on the functions cli to expose that. I have it logged as a separate work item - W-8070455